test/csuite/schema_abort/main.c

```diff
@@ -148,13 +147,23 @@ static void usage(void) WT_GCC_FUNC_DECL_ATTRIBUTE((noreturn));
 
 /*
  * usage --
- *     TODO: Add a comment describing this function.
+ *     Display usage statement and exit failure.
  */
 static void
 usage(void)
 {
-    fprintf(stderr, "usage: %s [-h dir] [-s stop_timestamp] [-T threads] [-t time] [-BClmvxz]\n",
+    fprintf(stderr, "usage: %s [-h dir] [-s stop_timestamp] [-T threads] [-t time] [-CSlmvxz]\n",
       progname);
+    fprintf(stderr, "%s",
```

Comment:
It seems that we should have a usage() function for all the arguments that are common to all the tests otherwise we will have duplicate code hard to maintain if those options change.

Reply:
Added them in c4dcf143c576738229475c665996d5e3835b018a